src/xo-app/home/index.js

```diff
@@ -216,6 +318,7 @@ const DEFAULT_TYPE = 'VM'
 
   return {
     areObjectsFetched,
```

Comment:
Should probably be moved to the `NoObjects_` component.

Check other data as well.